### PR TITLE
Added helper method for validating isomorphism in NamedGraphGenerator

### PR DESCRIPTION
**Pull request by tibrewalpratik17**
As a follow up to the conversation in #509 I have added a helper method to validate the isomorphism of generated NamedGraphs.

### CODE REVIEW

File: jgrapht-core/src/test/java/org/jgrapht/generate/NamedGraphGeneratorTest.java [hunk 1]
```diff
@@ -337,4 +326,19 @@ public void testThomsenGraph()
         assertEquals(diameter, gm.getDiameter(), 0.00000001);
         assertEquals(girth, GraphMetrics.getGirth(g), 0.00000001);
     }
+
+    private void validateIsomorphism( Graph<Integer, DefaultEdge> g, int value)
```
Comment (jsichi) [inline jgrapht-core/src/test/java/org/jgrapht/generate/NamedGraphGeneratorTest.java]: I suggest "validateAutomorphismCount" as a more accurate name.

**Comment by jsichi** [comment]
Good.  If you want to apply this to other existing named graph tests, go ahead and do it in this same pull request.

**Comment by tibrewalpratik17** [comment]
Okay... Great ...

**Comment by tibrewalpratik17** [comment]
@jsichi Though initially I thought of adding the validation for most of them but it was adding a huge overload while compiling so I restricted to validate only some of them.

**Comment by jsichi** [comment]
Sweet, thanks!

**Pull request merged**